.github/pull_request_template.md

```diff
@@ -5,11 +5,35 @@
 ## How
 *Describe the solution*
 
+## Recommended reading order
+1. `x.java`
+2. `y.python`
+
 ## Pre-merge Checklist
-- [ ] *Run integration tests*
-- [ ] *Publish Docker images*
+Expand the checklist which is relevant for this PR. 
+
+<details><summary> <strong> Connector checklist </strong> </summary>
+<p>
+
+- [ ] Issue acceptance criteria met
+- [ ] Unit & integration tests added as appropriate (and are passing)
+    * Community members: please provide proof of this succeeding locally e.g: screenshot or copy-paste acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
+- [ ] `/test` command documented [here]() is passing. 
+    * Community members can skip this, Airbyters will run this for you. 
+- [ ] Code reviews completed
+- [ ] Credentials added to Github CI if needed and not already present. [instructions for injecting secrets into CI](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#using-credentials-in-ci). 
+- [ ] Documentation updated 
+    - [ ] README
+    - [ ] CHANGELOG.md
+    - [ ] Reference docs in the `docs/integrations/` directory.
+    - [ ] If adding a new connector, update the `docs/SUMMARY.md` to point to the new docs in `docs/integrations/`
```

Comment:
I am working on #3867 on the side, which automatically takes care of the above two entries.

Reply:
wonderful. I'll remove this point then.